### ISSUE TITLE
Search packages

### ISSUE THREAD

**Issue by Seanld** [Seanld/Latte#7]
Need to add new functionality: the ability to search packages in your list by running a command like `latte search <package-name>` and it will scan all of your repositories in the cache for a repository of that name.

Adding this _may_ require having a package index file somewhere inside of the host server, unless there's some way to scan files on the host sever remotely _without_ needing a special file with all packages listed inside explicitly.

**Comment by wtpisaac** [comment]
This would be useful; another thing I'd love to see would be the ability to search with "*" to list all packages (may be useful for scripts that utilize Latte)

**Comment by Seanld** [comment]
@cyanisaac like `latte search *`? To list all packages that you can install based on your repositories?

**Comment by wtpisaac** [comment]
Yes. Alternatively some kind of list command. It could be useful for people exploring packages or useful for extension by scripts

**Comment by Seanld** [comment]
@cyanisaac I'm already working on this as we speak... Maybe I can close this issue before the end of the day!
